pkgs/applications/audio/cardinal/default.nix

```diff
@@ -21,22 +25,26 @@
 
 stdenv.mkDerivation rec {
   pname = "cardinal";
-  version = "22.11";
+  version = "22.12";
 
   src = fetchurl {
     url =
       "https://github.com/DISTRHO/Cardinal/releases/download/${version}/cardinal+deps-${version}.tar.xz";
-    sha256 = "sha256-xYQi209whY5/lN+6Fp7PTp7JSzL6RS6VL+Exst7RrS0=";
+    sha256 = "sha256-fyko5cWjBNNaw8qL9uyyRxW5MFXKmOsBoR5u05AWxWY=";
   };
 
   prePatch = ''
     patchShebangs ./dpf/utils/generate-ttl.sh
   '';
 
+  dontUseCmakeConfigure = true;
   enableParallelBuilding = true;
 
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [
+    cmake
```

Comment:
cmake and file are presumably nativeBuildInputs. set `strictDeps` to true if unsure.

Reply:
Thanks!